### ISSUE TITLE
Add license details for UKESM1-1-LL

### ISSUE THREAD

**Issue by matthew-mizielinski** [WCRP-CMIP/CMIP6_CVs#1071]
The first dataset for UKESM1-1-LL has made it onto ESGF with more to follow.

I'd like to update the corresponding entry with equivalent information as for UKESM1-0-LL;
```
            "license_info":{
                "exceptions_contact":"@metoffice.gov.uk <- cmip6.ukesm1",
                "history":"2022-05-04: initially published under CC BY-SA 4.0; 2021-11-15: relaxed to CC BY 4.0",
                "id":"CC BY 4.0",
                "license":"Creative Commons Attribution 4.0 International License (CC BY 4.0; https://creativecommons.org/licenses/by/4.0/)",
                "source_specific_info":"https://ukesm.ac.uk/licensing-of-met-office-nerc-and-niwa-cmip6-data/",
                "url":"https://creativecommons.org/licenses/by/4.0/"

```
Also need to set the cohort to `Published`

**Comment by durack1** [comment]
@matthew-mizielinski thanks for this. Should we amend the `history` attribute to:

```
"history":"2022-05-04: initially published under CC BY 4.0",
```

**Comment by matthew-mizielinski** [comment]
Hi @durack1 , I'm not sure -- the license text in the files is still the CC BY-SA license version as we are using the CVs built from version `6.2.56.11`. I see that work is starting on extending the license options in CMOR, but until that is completed and we've integrated any changes in CMOR into our workflow we'll be using the same license text as always. 

Also CEDA use PrePARE to validate files prior to publication, so I'd have to arrange the update to their publication system too.

The initial UKESM1-1-LL submission is going to be fairly limited (DECK plus a few hists and ssps), but should more be required in the future after the CMOR update we'll pick up the more relaxed license.

**Comment by durack1** [comment]
@matthew-mizielinski no problem, will amend the "relaxed" to the same date that data is published so we don't have any confusion (in my head - data has no license assigned if it doesn't exist yet)

**Comment by durack1** [comment]
Fixed by #1072
